manifests/s/Slackadays/Clipboard/0.9.0.1/Slackadays.Clipboard.installer.yaml

```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Slackadays.Clipboard
+PackageVersion: 0.9.0.1
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: bin\cb.exe
+  PortableCommandAlias: cb
+Installers:

```

Comment:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

Reply:
I guess I'll need to update it separately for both the archs separately. Will push a comment soon when I am back on my laptop.